cardinal/bot.py

```diff
@@ -99,9 +99,20 @@ def signedOn(self):
 
         # Set the currently connected network
         self.network = self.factory.network
-
+        self.network_alias = self.factory.network.split('.')[-2]
+
+        if self.network_alias == 'quakenet':
```

Comment:
If we add a condition for quakenet into the core, I'd rather have it toggle-able by a config option, than by hostname (geographically-specific hostnames could break on this, e.g.).

I also think we can send AUTH directly to the server through `cardinal.send()`.